pontoon/base/static/js/main.js

```diff
@@ -723,6 +723,21 @@ $(function() {
     }
   });
 
+  var clipboard = new Clipboard('.search .machinery li');
+
+  clipboard.on('success', function(event) {
+    var successMessage = $('<span class="clipboard-success">Copied!</span>'),
+        $trigger = $(event.trigger);
+
+    $('.clipboard-success').remove();
+    $trigger.find('header').prepend(successMessage);
+    setTimeout(function() {
+      successMessage.fadeOut(500, function() {
+         successMessage.remove();
+      });
+    }, 500);
```

Comment:
My instinct says it feels better if you change this value to 1000. But I have no scientific proof for that. ;)